includes/Modules/Idea_Hub.php

```diff
@@ -146,12 +161,126 @@ function( $maybe_empty, $postarr ) {
 	 */
 	public function register() {
 		$this->register_scopes_hook();
+		if ( $this->is_connected() ) {
+			/**
+			 * Show admin notices on the posts page if we have saved / new ideas.
+			 */
+			add_filter( 'googlesitekit_admin_notices', $this->get_method_proxy( 'admin_notice_idea_hub_ideas' ) );
+		}
 
 		$this->post_name_setting->register();
 		$this->post_text_setting->register();
 		$this->post_topic_setting->register();
 	}
 
+	/**
+	 * Shows admin notification for idea hub ideas on post list screen.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $notices Array of admin notices.
+	 * @return array Array of admin notices.
+	 */
+	private function admin_notice_idea_hub_ideas( $notices ) {
+		global $post_type;
+		$current_screen = get_current_screen();
+		if ( is_null( $current_screen ) || 'edit-post' !== $current_screen->id || 'post' !== $post_type ) {
+			return $notices;
+		}
+		$transients      = new Transients( $this->context );
+		$dismissed_items = new Dismissed_Items( $this->user_options );
+
+		$notices[] = new Notice(
+			'idea-hub_saved-ideas',
+			array(
+				'content'         => function() {
+					$saved_notice = array(
+						'text' => esc_html__( 'Need some inspiration? Revisit your saved ideas in Site Kit', 'google-site-kit' ),
+						'cta'  => esc_html__( 'See saved ideas', 'google-site-kit' ),
+						'link' => $this->context->admin_url() . '#new-ideas',
+					);
+					ob_start();
+					?>
+					<p>
+						<?php echo esc_html( $saved_notice['text'] ); ?>
+						<a href="<?php echo esc_url( $saved_notice['link'] ); ?>"
+						><?php echo esc_html( $saved_notice['cta'] ); ?></a>
+					</p>
+					<?php
+					return ob_get_clean();
```

Comment:
Let's render directly and get rid of the unnecessary `$saved_notice` variable.

Reply:
https://github.com/google/site-kit-wp/pull/3668/commits/850a96b2717cadb8ddfc0ee4119accb15f07b6c1